airbyte-integrations/connectors/destination-s3-data-lake/src/test-integration/resources/application-test.yaml

```diff
@@ -0,0 +1,7 @@
+airbyte:
+  destination:
+    core:
+      load-pipeline:
+        strategy: direct
+        input-parts: 1
```

Comment:
nit: these two properties should live in `src/main/resources/application-override.yaml`, since they also apply in production

(the record-batch-size-override shouldn't even be needed :/ that should already be set everywhere by this thing https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/test/util/IntegrationTest.kt#L306)